### ISSUE TITLE
Upgrade to latest Expo SDK 50 or 51

### ISSUE THREAD

**Issue by theblockstalk** [Tonomy-Foundation/Tonomy-ID#1058]
Note this is currently blocked by: https://github.com/digitalcredentials/jsonld-signatures/issues/9

**Comment by theblockstalk** [comment]
duplicate of https://github.com/Tonomy-Foundation/Tonomy-ID/issues/1232
